packages/manager/modules/nutanix/src/dashboard/controller.js

```diff
@@ -1,15 +1,34 @@
-import { GUIDE_URL } from './constants';
+import moment from 'moment';
+
+import { GUIDE_URL, LICENSE_REGISTRATION_ENDS_IN_DAYS } from './constants';
 
 export default class NutanixDashboardCtrl {
   /* @ngInject */
-  constructor($translate, coreConfig) {
+  constructor($translate, atInternet, coreConfig) {
     this.$translate = $translate;
     this.user = coreConfig.getUser();
+    this.atInternet = atInternet;
   }
 
   $onInit() {
     this.nutanixGuideUrl =
       GUIDE_URL.ALL_GUIDE[this.user.ovhSubsidiary] ||
       GUIDE_URL.ALL_GUIDE.DEFAULT;
   }
+
+  getLicenseRegistrationEndDate() {
+    return moment(this.serviceInfo.creation)
+      .add(moment.duration(LICENSE_REGISTRATION_ENDS_IN_DAYS, 'days'))
+      .format('DD/MM/YYYY');
+  }
+
+  isLicenceRegisterDateExpired() {
+    const diff = moment().diff(moment(this.serviceInfo.creation), 'days');
+    return (
+      diff >= 0 &&
+      diff <= LICENSE_REGISTRATION_ENDS_IN_DAYS &&
+      this.getTechnicalDetails.nutanixCluster.license.edition ===
+        'Personal license'
```

Comment:
Can you move this to constant ? It will be easy to modify it if it needs change.

Reply:
Updated everywhere as suggested. Thanks